docs/source/guide/IssuesHelper.rad

```diff
@@ -0,0 +1,9 @@
+;; (def chain-id "some-unique-name")
+
+;; (def make-comment
+;;   (fn [sk pk comment-id text]
+;;     (def msg (string-append chain-id (show comment-id) text))
```

Comment:
What's this file?

Reply:
It's a file with a helper to create valid comments in the `Issues.lrad` chain.

Reply:
But it's all commented out?